config/locales/server.en.yml

```diff
@@ -309,6 +308,8 @@ en:
     attributes:
       category:
         name: "Category Name"
+      topic:
+        title: 'Title'
```

Comment:
Yes. When user posts a random text in the title text field, AR will bounce as invalid in errors. A message will appear "Title <invalid message in another language>". The "Title" itself can not be translated if the locale file doesn't declares so. Here the `key` is the default path where AR is searching for the error message.